taipy/core/_repository/_filesystem_repository.py

```diff
@@ -189,10 +189,7 @@ def __filter_files_by_config_and_owner_id(
     ):
         try:
             files = filter(lambda f: config_id in f.name, self.dir_path.iterdir())
-            entities = map(
-                lambda f: self.__file_content_to_entity(self.__filter_by(f, filters)),
-                files,
-            )
+            entities = (self.__file_content_to_entity(self.__filter_by(f, filters)) for f in files)
```

Comment:
So no map object is returned any longer?

Reply:
it will still return an iterator, which we can still use to loop over like a map object